mfkdf2-web/package.json

```diff
@@ -54,6 +55,7 @@
     "ts-node": "^10.9.0",
     "tsconfig-paths": "^4.2.0",
     "tsx": "^4.19.0",
-    "typescript": "^5.6.2"
+    "typescript": "^5.6.2",
+    "mfkdf": "github:lonerapier/MFKDF#differential-tests"
```

Comment:
We should likely create at least a branch on the `multifactor/MFKDF` repo for safe keeping.

This is more my problem than yours.